Description: Fix spelling mistake and character omission
 The patch fixes a spelling mistake and inserts a dot to correct a statement
 about processed files in the documentation.
Author: Arno Onken <asnelt@asnelt.org>
Index: rrep-1.3.6/doc/rrep.texi
===================================================================
--- rrep-1.3.6.orig/doc/rrep.texi
+++ rrep-1.3.6/doc/rrep.texi
@@ -211,7 +211,7 @@
 @item -h
 @itemx --help
 @cindex help
-Display a help message that descibes the command line options and exit
+Display a help message that describes the command line options and exit
 afterwards.
 
 @item -i
@@ -355,7 +355,7 @@
 @samp{/home/user/hello} as well.
 However, files and directories starting with the @samp{.} character and binary
 files are ignored.
-Moreover, the @samp{--include=*c} option causes @command{rrep} to process only
+Moreover, the @samp{--include=*.c} option causes @command{rrep} to process only
 files with file names that end with @samp{.c}.
 
 
